app/src/main/java/com/sy/studentsapp/AddStudentActivity.kt

```diff
@@ -27,8 +27,8 @@ class AddStudentActivity : AppCompatActivity() {
             insets
         }
 
-        val saveButton: Button = findViewById(R.id.add_student_activity_save_button)
-        val cancelButton: Button = findViewById(R.id.add_student_activity_cancel_button)
```

Comment:
the standards is to put the Name_of_the_activity_Action_Type like was before:
add_student_activity_save_button
add_student - name of the activity
save - action
button - type